compiler/rustc_mir/src/interpret/step.rs

```diff
@@ -77,7 +77,7 @@ impl<'mir, 'tcx: 'mir, M: Machine<'mir, 'tcx>> InterpCx<'mir, 'tcx, M> {
     /// Runs the interpretation logic for the given `mir::Statement` at the current frame and
     /// statement counter. This also moves the statement counter forward.
     crate fn statement(&mut self, stmt: &mir::Statement<'tcx>) -> InterpResult<'tcx> {
-        info!("{:?}", stmt);
```

Comment:
I would really like to keep this at `info!` so that this level of logging is available for regular rustc builds.

Well really, I don't care what level it is at, as long as I can make rustc emit this without having to build a rustc with debug assertions enabled.